scripts/maintenance/check_consistency_data.py

```diff
@@ -0,0 +1,165 @@
+        f"database started: adminer available on http://127.0.0.1:18080/?pgsql=postgres&username={postgres_username}&db=simcoredb&ns=public"
+    )
+
+    # wait a bit that the dockers are up
+    # time.sleep(5)
+
+    db_file_paths = set()
+    # now the database is up, let's go through the projects table
+    with psycopg2.connect(
+        host="127.0.0.1",
+        port=5432,
+        database="simcoredb",
+        user=postgres_username,
+        password=postgres_password,
+    ) as conn:
+        with conn.cursor() as cursor:
+            cursor.execute(
+                "SELECT uuid, workbench"
+                ' FROM "projects"'
+                " INNER JOIN users"
+                " ON projects.prj_owner = users.id"
+                " WHERE users.role != 'GUEST'"
+            )
+            typer.echo(
+                f"found {cursor.rowcount} projects, now getting project/node ids..."
+            )
+            project_nodes = {}
+            rows = cursor.fetchall()
+            with typer.progressbar(rows) as progress:
+                for row in progress:
+                    project_uuid, workbench = row
+                    project_nodes[project_uuid] = list(workbench.keys())
+            typer.echo(
+                f"processed {cursor.rowcount} projects, now looking for files in the file_meta_data table..."
+            )
+
+            with typer.progressbar(project_nodes) as progress:
+                for project_uuid, node_ids in project_nodes.items():
+                    array = str([f"{project_uuid}/{n}%" for n in node_ids])
+
+                    if not node_ids:
+                        continue
+                    cursor.execute(
+                        "SELECT file_uuid"
+                        ' FROM "file_meta_data"'
+                        f" WHERE file_meta_data.file_uuid LIKE any (array{array}) AND location_id = '0'"
+                    )
+
+                    # here we got all the files for that project uuid/node_ids combination
+                    file_rows = cursor.fetchall()
+                    for file in file_rows:
+                        db_file_paths.add(file[0])
+                progress.update(1)
+
+    db_file_entries_path = Path.cwd() / "project_file_entries.txt"
+    db_file_entries_path.write_text("\n".join(db_file_paths))
+    typer.echo(
+        f"processed {len(project_nodes)} projects, found {len(db_file_paths)} file entries, saved in {db_file_entries_path}"
+    )
+
+    # ok now we need to connect to s3
+    typer.echo(
+        f"now connecting with S3 backend and getting file for {len(project_nodes)} projects..."
+    )
+    s3_file_entries = set()
+    with typer.progressbar(project_nodes) as progress:
+        for project_uuid in progress:
+            try:
+                completed_process = subprocess.run(
+                    f"docker run "
+                    f"--env MC_HOST_mys3='https://{s3_access}:{s3_secret}@{s3_endpoint}' "
+                    "minio/mc "
```

Comment:
for subprocess, why don't you pass arguments? see e.g. [this example](https://github.com/ITISFoundation/osparc-simcore/blob/32f6501d7336931f6458c4e0cadbabdab683f0db/packages/pytest-simcore/src/pytest_simcore/helpers/utils_docker.py#L1410)